examples/gcoap/README.md

```diff
@@ -53,6 +67,18 @@ CLI output:
     </>;title="General Info";ct=0,</time>;if="clock";rt="Ticks";title="Internal Clock";ct=0;obs,</async>;ct=0
 
 
+## Other available CoAP implementations and applications
+
+RIOT is compatible with other CoAP implementations with associated test
+applications:
+* [Nanocoap](../tests/pkg_nanocoap): a very lightweight CoAP server based on
```

Comment:
Requires an empty line before the list to format correctly in ReText. Formats OK in GitHub, so your choice.

Also, Nanocoap -> Microcoap

Reply:
I just tested these links, and the Nanocoap and Microcoap links don't work on GitHub. Need to pop up one more directory level.